js/utils.js

```diff
@@ -1596,6 +1596,35 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
   }
 
+  /**
+   * Function is formatting the input values to string
+   * @param {*} options Input values that can be of any type
+   * @returns The formatted input value into string value
+   */
+
+  function toFormattedString(options) {
+    switch (typeof options) {
+      case 'string':
+        return options;
+      case 'number':
+      case 'boolean':
+        return options.toString();
+      case 'object':
+        if (Array.isArray(options)) {
+          options = options.filter(function(item) {
```

Comment:
I think a better way is to use

```js
options = _.filter(_.map(options, toFormattedString), function(part) { return part.trim().length });
```

This does two things:

1. Each value in the array could be of any type. This treats each value in the array with the same logic as the other values.
2. `undefined` and `null`, which results in `""` is removed instead of outputting `, ,`

Reply:
@tonytlwu, fixed

Reply:
@tonytlwu, fixed